From bedec82615ca6858c3e9494650a5bbf33c710ebb Mon Sep 17 00:00:00 2001
From: Luca Boccassi <bluca@debian.org>
Date: Sun, 31 Oct 2021 12:29:14 +0000
Subject: [PATCH] Fix build with meson 0.60

Positional parameters to merge_file() were never allowed and always
ignored, so just drop it.
See: https://github.com/mesonbuild/meson/issues/9441

Fixes #97

Upstream-Status: Backport
Signed-off-by: Alexander Kanavin <alex@linutronix.de>
---
 data/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/data/meson.build b/data/meson.build
index 4987937..ed0e2c4 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -29,7 +29,6 @@ configure_file(
 policy = act_namespace.to_lower() + '.policy'
 
 i18n.merge_file(
-  policy,
   input: policy + '.in',
   output: policy,
   po_dir: po_dir,
